flink-connector-kinesis/pom.xml

```diff
@@ -256,6 +256,11 @@ under the License.
             <artifactId>netty-nio-client</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>apache-client</artifactId>
+        </dependency>
+
```

Comment:
Why is this needed explicitly in here and `aws-base`?

Reply:
Got it, removing this and keep it in aws-base